actions/matrixsynapse

```diff
@@ -80,6 +84,26 @@ def subcommand_disable(_):
     action_utils.service_disable('matrix-synapse')
 
 
+def subcommand_enable_registration(_):
+    """ Enable public registration"""
+    file_path = '/etc/matrix-synapse/homeserver.yaml'
```

Comment:
I would like it better if file_path was a module variable, e.g. `CONFIG_PATH = '/etc/matrix-synapse/homeserver.yaml'`, and re-use it everywhere - so if the path ever changes, there would be only a single changed line in the code.